vignettes/EnrichmentBrowser.Rmd

```diff
@@ -0,0 +1,989 @@
+
+For more information on experimental design, see the [limma user's
+guide](http://bioconductor.org/packages/limma), chapter 9.
+
+For the ALL dataset, the **GROUP** variable indicates whether the
+BCR-ABL gene fusion is present (1) or not (0).
+
+```{r sample-groups-ALL}
+ allSE$GROUP <- ifelse(allSE$mol.biol =="BCR/ABL", 1, 0) 
+ table(allSE$GROUP)
+```
+
+For the airway dataset, it indicates whether the cell lines have been treated 
+with dexamethasone (1) or not (0).
+
+```{r sample-groups-airway}
+airSE$GROUP <- ifelse(airway$dex == "trt", 1, 0)
+table(airSE$GROUP)
+```
+
+Paired samples, or in general sample batches/blocks, can be defined via
+a **BLOCK** column in the `colData` slot. For the airway dataset, the sample
+blocks correspond to the four different cell lines.
+
+```{r sample-blocks}
+airSE$BLOCK <- airway$cell 
+table(airSE$BLOCK)
+```
+
+For microarray expression data, the `deAna` function carries 
+out a differential expression analysis between the two groups based on 
+functionality from the `r Biocpkg("limma")` package.
+Resulting fold changes and *t*-test derived *p*-values for each gene are 
+appended to the `rowData` slot.
+
+```{r DE-ana-ALL}
+allSE <- deAna(allSE, padj.method = "BH")
+rowData(allSE)
+```
+ 
+Nominal *p*-values (`PVAL`) are corrected for multiple testing 
+(`ADJ.PVAL`) using the method from Benjamini and Hochberg implemented 
+in the function`p.adjust` from the `r CRANpkg("stats")` package.
+
+To get a first overview, we inspect the *p*-value distribution and the volcano 
+plot (fold change against *p*-value).
+
+```{r}
+par(mfrow = c(1,2))
+pdistr(rowData(allSE)$PVAL)
+volcano(rowData(allSE)$FC, rowData(allSE)$ADJ.PVAL)
+```
+
+The expression change of highest statistical significance is observed for the 
+ENTREZ gene 7525. 
+
+```{r DE-exmpl}
+ind.min <- which.min(rowData(allSE)$ADJ.PVAL) 
+rowData(allSE)[ind.min,]
+```
+
+This turns out to be the YES proto-oncogene 1
+([hsa:7525@KEGG](http://www.genome.jp/dbget-bin/www_bget?hsa:7525)).
+
+For RNA-seq data, the `deAna` function carries out a differential expression
+analysis between the two groups either based on functionality from `r Biocpkg("limma")`(that includes the `voom` transformation), or alternatively, the popular `r Biocpkg("edgeR")` or `r Biocpkg("DESeq2")` package.
+
+Here, we use the analysis based on `r Biocpkg('edgeR')` for demonstration.
+
+```{r DE-ana-airway}
+airSE <- deAna(airSE, de.method = "edgeR")
+rowData(airSE)
+```
+
+# ID mapping {#id:idmap}
+
+Using genomic information from different resources often requires
+mapping between different types of gene identifiers. Although primary
+analysis steps such as normalization and differential expression
+analysis can be carried out independent of the gene ID type, downstream
+exploration functionality of the `EnrichmentBrowser` is consistently based on NCBI Entrez Gene IDs. It is thus, in this regard, beneficial to initially map gene IDs of a different type to NCBI Entrez IDs.
```

Comment:
Make `EnrichmentBrowser` a link to the package.

Reply:
Okay